### PR TITLE
U1-79 Write unit test for action show-menu

### PR DESCRIPTION
**Pull request by huyhqit**
JIRA ticket: https://pass-career.atlassian.net/projects/U1/issues/U1-79

### CODE REVIEW

**Comment by knightus** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Coverage increased per file
===========================
+ source/actions/show-menu.js  57
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/pass-team/windows-10-lock-screen-image-extractor/pullRequest?prid=4990948&bid=16476416)

File: tests/unit/actions/show-menu.test.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+import randomDesktop from '../../../source/actions/random-desktop';
+
+jest.mock('caporal/lib/logger');
+jest.mock('enquirer/lib/prompts/select');
+jest.mock('../../../source/actions/get-images');
+jest.mock('../../../source/actions/show-settings');
+jest.mock('../../../source/actions/random-desktop');
+
+describe('Action show-menu', () => {
+  it('Should call action get-images when no item chôsen', async () => {
```
Comment (duyphaphach) [inline tests/unit/actions/show-menu.test.js]: chôsen: Typo error

Comment (duyphaphach) [reply tests/unit/actions/show-menu.test.js]: And the test case should be: "Should call get-images when no menu item chosen"

**Review by duyphaphach — APPROVED** [review]

**Comment by knightus** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Coverage increased per file
===========================
+ source/actions/show-menu.js  57
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/pass-team/windows-10-lock-screen-image-extractor/pullRequest?prid=4990948&bid=16476416)

**Review by duyphaphach — APPROVED** [review]

**Comment by knightus** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Coverage increased per file
===========================
+ source/actions/show-menu.js  57
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/pass-team/windows-10-lock-screen-image-extractor/pullRequest?prid=4990948&bid=16476416)

**Review by duyphaphach — APPROVED** [review]

**Pull request merged**